cmd/podman/restart.go

```diff
@@ -41,13 +40,14 @@ func init() {
 	flags.BoolVar(&restartCommand.Running, "running", false, "Restart only running containers when --all is used")
 	flags.UintVarP(&restartCommand.Timeout, "timeout", "t", define.CtrRemoveTimeout, "Seconds to wait for stop before killing the container")
 	flags.UintVar(&restartCommand.Timeout, "time", define.CtrRemoveTimeout, "Seconds to wait for stop before killing the container")
+	flags.BoolVarP(&restartCommand.AutoUpdate, "auto-update", "", false, "Restart all containers configured with an auto-update policy (experimental)")
```

Comment:
I don't see a reason to explicitly mark features as experimental. There is not policy around that in podman, right?

Reply:
> I don't see a reason to explicitly mark features as experimental. 

If a feature is likely to change in the near future, that's something we need to communicate to users. I do not want users to depend on the current behavior until we have gained some more experience and initial feedback.  Once we feel confident (and implemented roll backs), we can consider it stable.

> There is not policy around that in podman, right?

There's no formal policy, no. We did it a couple of times recently. API v2 would be on example, seccomp-policy for images another.